basic-sql-commands.md

```diff
@@ -40,8 +40,8 @@ SHOW DATABASES;
 +----------+
 | Database |
 +----------+
-| db1      |
-| system   |
+| "db1"    |
```

Comment:
@petermattis, is this true for table names, field names, and types, as well, since all of those now show up with quotes?

```
root@:15432> show columns from cockroaches;
+----------------+----------+-------+---------+
|     Field      |   Type   | Null  | Default |
+----------------+----------+-------+---------+
| "country_code" | "STRING" | false | NULL    |
| "count"        | "INT"    | true  | NULL    |
| "date"         | "DATE"   | true  | NULL    |
+----------------+----------+-------+---------+
root@:15432> show tables;
+---------------+
|     Table     |
+---------------+
| "cockroaches" |
+---------------+
```